mino/minogrpc/overlay.go

```diff
@@ -134,12 +171,12 @@ func (o overlayService) Stream(stream Overlay_StreamServer) error {
 			receiver.in <- msg
 		}
 	}()
-
+	o.handlerRcvr[apiURI[0]] = receiver
```

Comment:
I don't think this should exist. Everything should happen in the handler.Stream function.

Reply:
After reading, this should definitely be removed because a _Receiver_ is only an instance provided to the Stream handler, but it *must not* be transmitted. It has a given context and should close when this context is over. The same api URI must be able to manage several stream requests.